bitnami/airflow/values.yaml

```diff
@@ -120,8 +120,9 @@ airflow:
   ##
   cloneDagFilesFromGit:
     enabled: true
```

Comment:
Shouldn't this be false by default? Given that we are not providing a repo to pull from...